src/emqx_web_hook_actions.erl

```diff
@@ -292,8 +292,8 @@ add_default_scheme(URL) ->
     <<"http://", URL/binary>>.
 
 pool_opts(Params = #{<<"url">> := URL}) ->
-    #{host := Host0,
-      port := Port} = uri_string:parse(add_default_scheme(URL)),
+    #{host := Host0} = URIMap = uri_string:parse(add_default_scheme(URL)),
```

Comment:
#{host := Host, scheme := Scheme} = URIMap = uri_string:parse(add_default_scheme(URL)),
    Port = case Scheme of
        <<"https">> -> maps:get(port, URIMap, 443);
        _ -> maps:get(port, URIMap, 80)
    end,